hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -535,14 +535,20 @@ def move(encoded_region_name, server = nil)
     end
 
     #----------------------------------------------------------------------------------------------
-    # Merge two regions
-    def merge_region(region_a_name, region_b_name, force)
-      @admin.mergeRegionsAsync(
-        region_a_name.to_java_bytes,
-        region_b_name.to_java_bytes,
-        java.lang.Boolean.valueOf(force)
+    # Merge multiple regions
+    def merge_region(regions = [], force)
```

Comment:
Can we add support for the previous syntax? and not just when an array is used?

Reply:
Sure